features-json/font-family-system-ui.json

```diff
@@ -303,9 +303,10 @@
   },
   "notes":"",
   "notes_by_num":{
-    "1":"Supported as the `-apple-ui` value (only on Mac)"
+    "1":"Supported as the `-apple-ui` value (only on Mac)",
+    "2":"Supported as the 'BlinkMacSystemFont'  value (only on Mac)",
```

Comment:
Delete trailing comma (it's not legal in JSON).

Reply:
@awebdeveloper Can you fix this? Thanks.